javalanglib/src/main/scala/java/lang/System.scala

```diff
@@ -6,7 +6,35 @@ import scala.scalajs.js
 import js.Dynamic.global
 import scala.scalajs.LinkingInfo.assumingES6
 
+import java.{util => ju}
+import scalajs.runtime.environmentInfo
+
 object System {
+
+  private[this] var systemProperties = loadSystemProperties()
+
+  private def loadSystemProperties(): ju.Properties = {
+    val sysProp = new ju.Properties()
+    sysProp.setProperty("java.version", "1.8")
+    sysProp.setProperty("java.vm.specification.version", "1.8")
+    sysProp.setProperty("java.vm.specification.vendor", "Oracle Corporation")
+    sysProp.setProperty("java.vm.specification.name", "Java Virtual Machine Specification")
```

Comment:
See https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#getProperties%28%29

`java.vm.specification.version` is the version of the [JVMS](https://docs.oracle.com/javase/specs/) that is supported by this implementation.

`java.version` is the version of the JRE, i.e., the implementation itself. That is most usefully understood as the version of the JDK being implemented.